src/parastell.py

```diff
@@ -131,6 +145,14 @@ def construct_invessel_build(self, invessel_build):
         self.invessel_build.populate_surfaces()
         self.invessel_build.calculate_loci()
         self.invessel_build.generate_components()
+
+    def export_invessel_build(self, ivb_dict):
```

Comment:
I thought about doing the latter, but it is less robust against making changes to the content of the dictionary.

I'm comfortable with leaving it as is since the construct methods and export methods then have similar signatures and I think it makes it pretty easy to follow.